components/vectors/CheckIcon.tsx

```diff
@@ -11,7 +11,7 @@ export default function CheckIcon({
     <Svg
       width={width}
       height={height}
-      viewBox={`0 0 ${width} ${height}`}
+      viewBox={`0 0 32 32`}
```

Comment:
Since this icon is used in many places, it's not recommended to change it internally

Reply:
Since the viewBox defines the internal grid (coordinate system) used to draw the SVG, it's generally recommended to keep it as is to maintain the original design. And if you want to change the size of the SVG, just adjust the width and height attributes.

In our case, changing the viewBox along with the width and height was changing the box grid of the SVG instead of resizing it. That's why I decided to fix it by keeping the viewBox constant.